hq/app/logging/Cloudwatch.scala

```diff
@@ -41,25 +59,28 @@ object Cloudwatch extends Logging {
   def logAsMetric[T](data: Map[AwsAccount, Either[FailedAttempt, List[T]]], dataType: DataType.Value ) : Unit = {
     data.toSeq.foreach {
       case (account: AwsAccount, Right(details: List[T])) =>
-        putMetric(account, dataType, details.length)
+        putAwsMetric(account, dataType, details.length)
       case (account: AwsAccount, Left(_)) =>
         logger.error(s"Attempt to log cloudwatch metric failed. Data of type ${dataType} is missing for account ${account.name}.")
     }
   }
 
-  def putMetric(account: AwsAccount, dataType: DataType.Value , value: Int): Unit = {
-    val dimension = List(
-      (new Dimension).withName("Account").withValue(account.name),
-      (new Dimension).withName("DataType").withValue(dataType.toString)
-    )
+  def putAwsMetric(account: AwsAccount, dataType: DataType.Value , value: Int): Unit = {
+    putMetric("SecurityHQ", "Vulnerabilities", Seq(("Account", account.name),("DataType", dataType.toString)), value)
+  }
+
+  def putGcpMetric(project: String, dataType: DataType.Value , value: Int): Unit = {
+    putMetric("SecurityHQ", "Vulnerabilities", Seq(("GcpProject", project),("DataType", dataType.toString)), value)
+  }
 
-    val datum = new MetricDatum().withMetricName("Vulnerabilities").withUnit(StandardUnit.Count).withValue(value.toDouble).withDimensions(dimension.asJava)
-    val request = new PutMetricDataRequest().withNamespace("SecurityHQ").withMetricData(datum)
+  def putMetric(namespace: String, metricName: String, metricDimensions: Seq[(String, String)] , value: Int): Unit = {
```

Comment:
it might be a good idea to make this functoin `private` to discourage future devs from calling it directly wiith their own custom dimensions - it might be good to stick to the two AWS/GCP options

Reply:
Good point!